src/client/Client.js

```diff
@@ -38,7 +38,7 @@ class Client extends BaseClient {
     let data = process.env;
     try {
       // Test if worker threads module is present and used
-      data = require('worker_threads').workerData || data;
+      data = require('worker_threads').workerData ?? data;
     } catch {
       // Do nothing
     }
```

Comment:
I'm pretty sure we can remove the try/catch, worker_threads are available since 10.5.0 and are currently stable (from far before Node.js v14).

@vladfrangu thoughts?

Reply:
Yeah, we can totally remove the try/catch, due to the reasons you already specified